김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/view/adapter/BindingAdapter.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.mvvmstudy.view.adapter
+
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.example.mvvmstudy.R
+
+@BindingAdapter("imageUrl")
+fun loadImage(
+    imageView: ImageView,
+    url: String
+) {
+    Glide.with(imageView.context)
+        .load(url)
+        .placeholder(ContextCompat.getDrawable(imageView.context, R.drawable.ic_baseline_storage))
+        .error(ContextCompat.getDrawable(imageView.context, R.drawable.ic_baseline_storage))
+        .apply(RequestOptions.fitCenterTransform())
+        .into(imageView)
+}
```

Comment:
binding에서가 아닌 코드 로직 단계에서도 범용적으로 사용될 수 있을 것 같아서
확장함수로 만들어도 좋을 것 같아요
```suggestion
fun ImageView.loadImage(
    url: String
) {
    Glide.with(context)
        .load(url)
        .placeholder(ContextCompat.getDrawable(context, R.drawable.ic_baseline_storage))
        .error(ContextCompat.getDrawable(context, R.drawable.ic_baseline_storage))
        .apply(RequestOptions.fitCenterTransform())
        .into(this)
}
```

Reply:
오.. 끄적 끄적 메모

Reply:
끄적끄적.. 메모!